subprojects/docs/src/docs/userguide/basePlugin.adoc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[base_plugin]]
+== The Base Plugin
+
+The Base Plugin provides features that are common to most builds and adds a structure to the build that promotes consistency in how they are run. Its most significant contribution is a set of <<sec:base_tasks,'lifecycle' tasks>> that act as an umbrella for the more specific tasks provided by other plugins and build authors.
+
+[[sec:base_plugin_usage]]
+=== Usage
+
+----
+plugins {
+    id 'base'
+}
+----
+
+[[sec:base_tasks]]
+=== Tasks
+
+`clean` — type: `Delete`::
+Deletes the build directory and everything in it, i.e. the path specified by the `buildDir` project property.
```

Comment:
Can you add an API link to `buildDir`?

Reply:
I agree that the rendered result is not as good looking. However, I'd rather have readers understand where the property comes from by linking to the API.